webapp/src/index.js

```diff
@@ -43,6 +44,11 @@ class Plugin {
             );
         }
 
+        registry.registerWebSocketEventHandler(
+            `custom_${pluginId}_meeting_started`,
+            handleMeetingStarted(),
+        );
```

Comment:
We should test this with multiple tabs open at the same time

Reply:
This was a little confusing at first. Maybe we can have `handleMeetingStarted` itself accept the websocket message, then this can be written this way:

```suggestion
        registry.registerWebSocketEventHandler(
            `custom_${pluginId}_meeting_started`,
            handleMeetingStarted,
        );
```